Cargo.toml

```diff
@@ -29,6 +29,7 @@ sha2 = "0.10"
 telegram_types = "0.6.0"
 unicode-width = "0.1"
 url = "2.1.0"
+phf = { version = "0.11.0", features = ["macros"] }
```

Comment:
The list is otherwise sorted, so it would be good for this to be just after `parking_lot`.